doc/source/whatsnew/v1.5.0.rst

```diff
@@ -286,7 +286,7 @@ Missing
 
 MultiIndex
 ^^^^^^^^^^
--
+- Bug in :meth:`IndexOpsMixin.to_numpy` where multiindexed Series could not be converted to numpy arrays when an ``na_value`` was supplied (:issue:`45774`)
```

Comment:
the note would need to be user facing

Reply:
What do you mean by this comment? I do not understand completely what "needs to be user facing" means. Thanks.

Reply:
Hey @jreback . May you please elaborate on your comment.

Reply:
``IndexOpsMixin`` is private, this is not exposed through the api

Reply:
Comment should be addressed.